system_test/aest_sync_SUITE.erl

```diff
@@ -38,15 +38,15 @@
     peers   => [old_node2],
     backend => aest_docker,
     % Change to a compatible fixed version when possible
```

Comment:
So the comment can be deleted or updated?

Reply:
I'll do that right after the standup